nemo_curator/download/commoncrawl.py

```diff
@@ -383,6 +412,10 @@ def download_common_crawl(
         after the starting snapshot.
       output_type: The file type to save the data as.
       algorithm: A JusTextExtractor or ResiliparseExtractor object.
+      stop_lists: A dictionary stop lists, where the keys are languages (e.g., "ENGLISH")
+        and the values are Python frozensets denoting the list of stop words for that language.
+        If None, it defaults to jusText's stop lists: https://github.com/miso-belica/jusText/tree/main/justext/stoplists,
+        with added Thai, Chinese, and Japanese support.
```

Comment:
While I think it's important for NeMo Curator to support Thai, Chinese, and Japanese, I also think it would be a good idea for us to allow users to pass in their own stop lists as a workaround.

This way, if a language is not already supported, the user can do it themselves. Additionally, a user might not like the stop lists provided by jusText and want to pass in their own custom stop lists for that reason, too.